src/server/api/schema.js

```diff
@@ -444,6 +444,26 @@ const rootMutations = {
       }
       return organization
     },
+    assignUserToCampaign: async (_, { campaignId }, { user, loaders }) => {
+      let campaign
```

Comment:
Joining an organization requires the uuid, which is at some level the security layer for this api.  So, besides `campaignId` we need an argument `organizationUuid` (just like with joinOrganization) which should be tested against the campaign's organization.